### PR TITLE
Update user-roles.md

### PR DESCRIPTION
**Pull request by yujiaoMSFT**
Docs is not reflecting UI correctly.
When you check RBAC UI in M365D portal, it says "Manage security settings in Security Center", NOT "Manage security settings in Defender for Cloud".

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/user-roles.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ The following steps guide you on how to create roles in Microsoft 365 Defender.
     > [!NOTE]
     > This setting is only available in the Microsoft Defender for Endpoint administrator (default) role.
 
-- **Manage security settings in Defender for Cloud** - Configure alert suppression settings, manage folder exclusions for automation, onboard and offboard devices, and manage email notifications, manage evaluation lab
+- **Manage security settings in Security Center** - Configure alert suppression settings, manage folder exclusions for automation, onboard and offboard devices, and manage email notifications, manage evaluation lab
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/user-roles.md]:
```suggestion
- **Manage security settings in Security Center** - Configure alert suppression settings, manage folder exclusions for automation, onboard and offboard devices, manage email notifications, and manage evaluation lab
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @yujiaoMSFT, kindly commit the minor additional suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@yujiaoMSFT Gentle reminder, Thanks

**Comment by yujiaoMSFT** [comment]
@JohanFreelancer9 , @yogkumgit 
Thanks for the fix and sorry for my slow respond.
I checked updated text and looks good to me.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]
Approved with my suggested change committed. Unfortunately, I cannot commit the change.

**Pull request merged**